service-base-utils-bld-os/build.properties

```diff
@@ -0,0 +1,19 @@
+#  ************************************** #
+#  (C) Copyright IBM Corp. 2001, 2020     #
+#  SPDX-License-Identifier: Apache-2.0    #
+#  ************************************** #
+
+# Name of project
+projectGitProj=
+projectGitRepo=
+projectName=
+slackChannel=
+
+# Servers
+binariesURL=
+
+# This is a list of projects that are part of RTC but are not part of the build yet.
+# This is used in CSAR, Analysis ..etc to exclude these projects from the scan.

```

Comment:
Not sure what this file is for, but given the reference to RTC (?) and CSAR, might be worth getting rid of unless needed?

Reply:
Confirmed I can get rid of this file as well. Will remove.